src/components/LeaderboardRanking.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="d-flex ranking w-100 mt-1">
-    <div class="d-flex">
+    <div class="d-flex min-0">
```

Comment:
There's actually a Tailwind class `min-w-0` that already exists for this:
https://tailwindcss.com/docs/min-width